test/store.miniflare.test.ts

```diff
@@ -0,0 +1,91 @@
+/**
+ * @jest-environment miniflare
+ * @jest-environment-options {"scriptPath": "src/index.ts"}
```

Comment:
we don't actually have a worker script, but we need a file that can be read from local disk and this one seems good enough